### PR TITLE
Stops gibtonite (and colossus) from making lava.

### PR DESCRIPTION
**Pull request by PerishedFraud**
## About The Pull Request

Stops gibtonite from making lava on explosion (and colossus too) by making the lavaland floor indestructible.
The floor can still be deconstructed and dug into, which are the only other interactions I know.
Players are not able to obtain this floor tile, so while a more specialized fix might be possible, this works fine and is a whopping 1 line of code, easy to revert if needed later.

## Why It's Good For The Game

Gibtonite was designed as a fairly dangerous ore that can also help you mine faster. The turf damage change turned it into an annoying hazard with no upside, as it would destroy the floor and create lava which would:
-kill you immediately even if you'd survive otherwise
-smelt and then destroy any ores instead of mining them
-leave the patch of lava for the rest of the round until RCD'd

TLDR: Bad and unfun.

Colossus would also destroy the floor with his bolts and even melee attack though this is rarer. Clearly unintended.

## Testing Photographs and Procedure
<details>
<summary>Screenshots&Videos</summary>

yea

</details>

## Changelog
:cl:
tweak: lavaland floor is indestructible
/:cl:

### CODE REVIEW

**Review by PowerfulBacon — CHANGES_REQUESTED** [review]
Don't make it indestructible. Increase the health so that it can survive a non-devestating explosion.

**Comment by PerishedFraud** [comment]
> Don't make it indestructible. Increase the health so that it can survive a non-devestating explosion.

Testing revealed that gibtonite would still leave lava behind (notably in the tile it explodes in) unless the value was set to something ridiculous. While this would be better than what we have right now, it seemed like a halfbaked fix if it still left lava. The floor can still be deconstructed.

rks mentioned PREVENT_CONTENTS_EXPLOSION_1 earlier. Would that work as an alternative since it would allow other means of destruction?

**Comment by Rukofamicom** [comment]
I'd like to chime in that having the gibdonite leave a single lava tile would actually be much worse because, similar to single tile chasms, they're fairly hard to see and walking even diagonally over them causes you to suffer the consequences of entering the tile. 

To reconcile Bacon's request, simply remove the devastation damage from the gibtonite explosion - it can keep the same range and power (for the purposes of mining) this only removes its ability to instantly gib when placed on someone, as well as its ability to instantly break most tiles.

**Comment by PowerfulBacon** [comment]
No janky flags should be set on destructible turfs since it will lead to weird and inconsistent behaviour.

**Comment by PerishedFraud** [comment]
alright, stay tuned.

**Comment by MarkusLarsson421** [comment]
> Testing revealed that gibtonite would still leave lava behind (notably in the tile it explodes in)

I like this. It will still leave some lava behind but also mine the resources. Any resources it does mine is pushed back from the lava.

**Comment by PerishedFraud** [comment]
It's back.

**Comment by Rukofamicom** [comment]
> alright, stay tuned.

Please address the requested changes, this hasn't seen updates for close to three months. 
If this isn't stale or you want to re-open it after it auto-closes let us know.
